internal/test/factory.go

```diff
@@ -77,6 +77,18 @@ func GenerateBlobTxs(numTxs, blobsPerPfb, blobSize int) [][]byte {
 	return txs
 }
 
+func GenerateBlobs(blobSizes ...int) []*blob.Blob {
+	blobs := make([]*blob.Blob, len(blobSizes))
+	var err error
+	for i, size := range blobSizes {
+		blobs[i], err = blob.New(namespace.RandomBlobNamespace(), RandomBytes(size), shares.ShareVersionZero, nil)
```

Comment:
[optional follow up]: it could be useful for tests to generate blobs of share version 0 and share version 1 so that we catch edge cases caused by the signer.

Reply:
I think for some tests I will want to go for table like testing where both v1 and v0 are convered

Reply:
SGTM can resolve.